src/main/webapp/src/app/hidden-gem.service.ts

```diff
@@ -20,4 +20,18 @@ export class HiddenGemService {
   handleError(error: HttpErrorResponse) {
     return throwError('A data error occured, please try again.');
   }
+
+  findHiddenGemReccomendation(data: any) : Observable<HiddenGem[]> {
```

Comment:
Nit - seems like there is a typo in "Recommendation"

Reply:
Done Thanks 😊 . I'm bad at spelling